pkgs/pkgs-lib/formats.nix

```diff
@@ -1125,4 +1125,29 @@ optionalAttrs allowAliases aliases
     else
       throw "pkgs.formats.xml: Unknown format: ${format}";
 
+  plist =
```

Comment:
I think the format looks good.

Could you add docs to `nixos/doc/manual/development/settings-options.section.md`?